dev/io.openliberty.microprofile.telemetry.1.0.internal/src/io/openliberty/microprofile/telemetry/internal/rest/package-info.java

```diff
@@ -10,7 +10,8 @@
  * Contributors:
  *     IBM Corporation - initial API and implementation
  *******************************************************************************/
-@TraceOptions(traceGroup = "TELEMETRY")
+@TraceOptions(traceGroup = "TELEMETRY", messageBundle = "io.openliberty.microprofile.telemetry.common.resources.MPTelemetry")
+@org.osgi.annotation.versioning.Version("1.0")
```

Comment:
At the moment, the version is declared in the bnd.bnd. We should either declare it here or there, but not in both places.